src/components/containers.js

```diff
@@ -135,3 +137,34 @@ export const IndicatorsContainer = (props: IndicatorContainerProps) => {
     </Div>
   );
 };
+
+// ==============================
+// Value Spacer
+// ==============================
+
+export type ValueSpacerProps = CommonProps & {
+  /** Set when the value container should hold multiple values. This is important for styling. */
+  values: any,
+};
+export const valueSpacerCSS = () => ({
+  display: 'flex ',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+});
+export const ValueSpacer = (props: ValueSpacerProps) => {
+  const { children, values } = props;
+  return (
+    <Div
+      css={getStyles('value-spacer', props)}
+      {...innerProps}
+    >
+      <div style={{ margin: 0, padding: 0, height: 0, visibility: 'hidden' }}>
+        {values.placeholder && <Placeholder>{values.placeholder}</Placeholder>}
+        {values.options && values.options.length && values.options.map(
+          option => <SingleValue>{option}</SingleValue>
+        )}
```

Comment:
We then fill the hidden element with all the possible values (including the placeholder), which will force width to the widest element.

Since they use the `<Placeholder>`/`<SingleValue>` components, the style should be identical to that of the actual visibly selected element, meaning no width jumping!